lib/resty/timerng/init.lua

```diff
@@ -387,11 +384,9 @@ function _M:named_at(name, delay, callback, ...)
     assert(type(delay) == "number", "expected `delay to be a number")
     assert(delay >= 0, "expected `delay` to be greater than or equal to 0")
 
-    if delay >= self.max_expire
-        or (delay ~= 0 and delay < self.opt.resolution)
+    if delay >= self.max_expire or
```

Comment:
```suggestion
    if (delay >= self.max_expire) or
```
align to the next line.